manifests/params.pp

```diff
@@ -17,21 +17,28 @@
   $version = 'latest'
   $service = 'jenkins'
   $install_from_git = false
+  $install_from_pip = true
+  $install_from_pkg = false
   $git_revision     = 'master'
   $git_url          = 'https://git.openstack.org/openstack-infra/jenkins-job-builder'
 
   case $::osfamily {
     'RedHat', 'Amazon': {
-      $python_packages = [ 'python', 'python-devel', 'python-pip', 'python-argparse']
+      $python_packages = [ 'python', 'python-devel', 'python-pip', 'python-argparse', 'PyYAML']
```

Comment:
Yeah, looks like argparse is in the stdlib for 2.7 => https://docs.python.org/2.7/library/argparse.html#module-argparse